### DISCUSSION TITLE
in mongodb, update multiple deeps attribute

### DISCUSSION THREAD

**Discussion by kpouters** [mikro-orm/mikro-orm#4084] [Q&A]
const user = this.em.findOne({},User);

user.login.date = new Date();

this.em.persist(user);
this.em.flush();

-----------------------------
it is not work
is it normal?

**Comment by B4nan** [comment]
> const user = this.em.findOne({},User);

This should be `em.findOne(User, cond)` and the condition is required, passing just `{}` would fail.

> this.em.flush();

And this needs to be awaited.

If it fails, provide some details, e.g. how it fails, what error you get, including the stack trace.

If it is not doing anything, you are most probably using wrong EM context.
